doc/Type/Proc/Async.pod6

```diff
@@ -14,28 +14,33 @@ moment.
 C<Proc::Async> allows you to run external commands asynchronously, capturing
 standard output and error handles, and optionally write to its standard input.
 
-    # command with arguments
-    my $proc = Proc::Async.new('echo', 'foo', 'bar');
-
-    # subscribe to new output from out and err handles:
-    $proc.stdout.tap(-> $v { print "Output: $v" }, quit => { say 'caught exception ' ~ .^name });
-    $proc.stderr.tap(-> $v { print "Error:  $v" });
-
-    say "Starting...";
-    my $promise = $proc.start;
-
-    # wait for the external program to terminate
-    await $promise;
-    say "Done.";
-
-This produces the following output:
-
-=begin code :skip-test
-Starting...
-Output: foo bar
-Done.
+=begin code :skip-test<io>
+    my $line-source = Proc::Async.new("perl6", "bin/line-source.p6", :w);
+
+    react {
+         whenever $line-source.stdout.lines -> $l is copy {
+             put $l.chop;
+         }
+         whenever $line-source.start { 
+             say "exitcode: ", .exitcode;
+             done;
+         }
+         whenever signal(SIGINT) {
+             $line-source.close-stdin;
+             exit 0;
```

Comment:
This exits without killing the process. In some cases the process can go down if it writes to stdin and the pipe is broken, but generally this is incorrect.